.github/workflows/foreman.yml

```diff
@@ -125,6 +125,18 @@ jobs:
       - name: Run rake ${{ matrix.task }}
         run: bundle exec rake ${{ matrix.task }}
 
+  katello:
+    name: Katello
+    needs:
+      - rubocop
+    uses: theforeman/actions/.github/workflows/foreman_plugin.yml@v0
+    with:
+      plugin: katello
+      plugin_repository: Katello/katello
+      foreman_version: ${{ github.ref }}
+      postgresql_container: ghcr.io/theforeman/postgresql-evr
+      test_existing_database: false
+
   result:
```

Comment:
should this depend on `katello`?

Reply:
In what way?

Reply:
In the way that the dummy "result" test that we use as a branch protection rule (so that we do not have to rely on the more dynamic matrix based results) should depend on the newly added katello test, thus making it blocking for merges.

Reply:
added now and rebased.